.github/workflows/scheduled.yml

```
@@ -990,3 +992,9 @@ jobs:
             /tmp/writer_fuzzer_repro
             /tmp/server.log
             /var/log
+
+  linux-clang:
+    name: Build with Clang
```

Like this it will also run on [PRs](https://github.com/facebookincubator/velox/actions/runs/11187830220/job/31105594743?pr=10767) we want to add an if:
```suggestion
  linux-clang:
    if: ${{ github.event_name == 'schedule' }}
    name: Build with Clang
```